coalib/bearlib/abstractions/Linter.py

```diff
@@ -447,10 +447,13 @@ def run(self, filename, file, **kwargs):
                     FunctionMetadata.filter_parameters(
                         self._get_create_arguments_metadata(), kwargs))
 
-                arguments = (self.get_executable(),) + tuple(
-                    self.create_arguments(
-                        filename, file, config_file,
-                        **create_arguments_kwargs))
+                args = self.create_arguments(filename, file, config_file,
+                                             **create_arguments_kwargs)
+
+                if args is None:  # pragma: no cover
+                    self.warn("The tuple of arguments cannot be None.")
```

Comment:
using try-catch has the advantage that you can return any sequence, you can even use `yield` then which is very cool and convenient sometimes (maybe some bears shall actually use them...)